conversational-client/src/app/components/agents/history-tutor/history-tutor.component.ts

```diff
@@ -1,26 +1,38 @@
 import {Component} from '@angular/core';
+import {Agent, AgentState} from '@data/conversation';
+
+interface WorldState {
+  questions: Question[];
+}
+
+interface Question {
+  question: string;
+  answers: Answer[];
+}
+
+interface Answer {
+  answer: string;
+  hasAnswered: boolean | string;
+  has_answered: boolean | string;
```

Comment:
you may as well remove has_answered while you're here -- it was a hack since the format returned was flaky, but we need to find a better strategy.

(don't worry about breaking history_tutor, I think I already broke it and need to sort it out next wk)

Reply:
I'll let you deal with history tutor, since this PR is already large :)